values.yaml

```diff
@@ -63,6 +63,15 @@ config:
     url: 'https:\/\/helx.renci.org'
   workspaces:
     enabled: "true"
+  analytics: >-
+    {
+      "enabled": true,
+      "platform": "mixpanel",
+      "auth": {
+        "mixpanel_token":  "0c39f8410fd548bfa1976957d0248289",
```

Comment:
Yeah, I believe tracking ids are generally public. If someone wanted to find the tracking id, it wouldn't require much more effort even if we were to make the tokens better protected.

Generally, you go into the project settings on the analytics platform (GA or Mixpanel) and set it so that data can only be sent from specific domains. Still, there's no way to stop someone from sending garbage data from the site itself, but I don't think that's really an area of concern.